third-party/easyloggingpp/src/easylogging++.h

```diff
@@ -2750,6 +2763,8 @@ class AsyncDispatchWorker : public base::IWorker, public base::threading::Thread
   std::condition_variable cv;
   bool m_continueRunning;
   base::threading::Mutex m_continueRunningLock;
+  std::thread m_t1;
```

Comment:
1. Rename `m_t1` into `m_workerThread`
2. Should probably go into #IFDEF ASYNC...

Reply:
renamed, the whole class is within an ifdef statement